### PR TITLE
themev1tov2: Adapt to new model of layergroup_treeitem

### LINKED ISSUE #3001: Problem running script themev1tov2
Version 2.2.0rc2

When running script .build/venv/bin/themev1tov2, I get the following error. What is the problem ? 

Emptying table main.layer_wmts.
Emptying table main.layer_wms.
Emptying table main.ogc_server.
Converting layerv1.
Traceback (most recent call last):
  File ".build/venv/bin/themev1tov2", line 11, in <module>
    sys.exit(main())
  File "/var/www/vhosts/cartoriviera_preprod/private/cartoriviera_geomapfish/.build/venv/local/lib/python2.7/site-packages/c2cgeoportal/scripts/themev1tov2.py", line 103, in main
    layer_v1tov2(session, layer)
  File "/var/www/vhosts/cartoriviera_preprod/private/cartoriviera_geomapfish/.build/venv/local/lib/python2.7/site-packages/c2cgeoportal/scripts/themev1tov2.py", line 218, in layer_v1tov2
    new_link.group = link.group
AttributeError: 'LayergroupTreeitem' object has no attribute 'group'

### PR DESCRIPTION
**Pull request by ger-benjamin**
FIX #3001 

Any idea of what that was @sbrunner ?
(must be also reported in 2.2)

### CODE REVIEW

**Comment by yjacolin** [comment]
I tested this changes and I get another issue (probably not linked to this issue, though). Layers are not linked to groups. I used this script for a GMF 2.1 migration with success. When migrate the custoemr database after migration to GMF 2.2 the script doesn't seems to work correctly.

**Comment by ger-benjamin** [comment]
Looks good now (the layergroup_treeitem model was changed). @yjacolin can you test ?

**Comment by ger-benjamin** [comment]
@yjacolin work only for 2.2 I'll do another PR for 2.1 (in 2.1, `treegroup` is `group` and `treeitem` is `item`)

**Comment by ger-benjamin** [comment]
Looks good now (the layergroup_treeitem model was changed). @yjacolin can you test ?

File: c2cgeoportal/scripts/themev1tov2.py [hunk 1]
```diff
@@ -215,8 +215,8 @@ def layer_v1tov2(session, layer):
         new_link.ordering = link.ordering
         new_link.description = link.description
         new_link.treegroup_id = link.treegroup_id
```
Comment (ger-benjamin) [inline c2cgeoportal/scripts/themev1tov2.py]: @sbrunner is this line really useful ? it looks like it has no effect

**Comment by yjacolin** [comment]
@ger-benjamin it works correctly now! Thanks.

File: c2cgeoportal/scripts/themev1tov2.py (hunk 1 above)
Comment (sbrunner) [inline c2cgeoportal/scripts/themev1tov2.py]: Effectively that's looks strange, can you try to remove it ad do a test?

Comment (ger-benjamin) [inline c2cgeoportal/scripts/themev1tov2.py]: Already done before, and I've see no changes

Comment (sbrunner) [inline c2cgeoportal/scripts/themev1tov2.py]: Then remove it :-)

Comment (ger-benjamin) [inline c2cgeoportal/scripts/themev1tov2.py]: @sbrunner Done, can I merge it ?

**Pull request merged**